frontend/src/components/admin/WikidataCacheSection.tsx

```
@@ -0,0 +1,310 @@
+        <Stack direction="row" alignItems="center" justifyContent="space-between" sx={{ mb: 1 }}>
+          <Button size="small" startIcon={<ExpandLessIcon />} onClick={() => setOpen(false)}>
+            Wikidata cache
+          </Button>
+          <Button
+            size="small"
+            color="warning"
+            startIcon={<DeleteSweepIcon />}
+            disabled={busy || kinds.every(k => k.entries === 0)}
+            onClick={() => clear.mutate(undefined)}
+          >
+            Clear this source's cache
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)